### DISCUSSION TITLE
Claude Computer Use

### DISCUSSION THREAD

**Discussion by SerenityNrrd** [coollabsio/coolify#4005] [Service Template Requests]
Could someone smarter then me figure out how to get this to work in docker on Coolify. i have it set up and i have tried to get it to work the same way i have gotten a few others to work but i can't figure this out one. 

https://github.com/anthropics/anthropic-quickstarts/tree/main/computer-use-demo

**Comment by Skeyelab** [comment]
@SerenityNrrd - this might be of use to you:

https://it-tools.tech/docker-run-to-docker-compose-converter

**Comment by SerenityNrrd** [comment]
oh that's awesome.. .Thanks
